activiti-process-runtime-api/activiti-process-runtime-api/src/main/java/org/activiti/runtime/api/model/VariableInstance.java

```diff
@@ -16,6 +16,14 @@
 
 package org.activiti.runtime.api.model;
 
-public interface ProcessInstance {
+public interface VariableInstance {
```

Comment:
Not sure about the naming here: `VariableInstance` or just `Variable`?

Reply:
I'm ok wiht VariableInstance.. it is quite term common in BPM.